src/site-server.ts

```diff
@@ -180,6 +188,14 @@ export class SiteServer {
 
 		const wpCliArgs = parse( args );
 
+		if ( ! includePlugins ) {
+			wpCliArgs.push( '--skip-plugins' );
+		}
+
+		if ( ! includeThemes ) {
+			wpCliArgs.push( '--skip-themes' );
+		}
```

Comment:
I'm adding some parameters in case we can include the plugins when executing some specific wp-cli command, but maybe we could avoid those parameters and directly add the arguments:

```
wpCliArgs.push( '--skip-plugins' );
wpCliArgs.push( '--skip-themes' );
```

Reply:
I also thought in adding this arguments at `execute-wp-cli` level, but that way we would adding the arguments also when we execute wp-cli from the CLI. 

https://github.com/Automattic/studio/blob/2c27eaf3b26045eca0545ca577b3a72b56544636/vendor/wp-now/src/execute-wp-cli.ts#L73

Reply:
It fixes the issue with the plugin that was leaking PHP warnings to WP CLI commands output.

The only part I'm not convinced of is the default value. The `executeWpCliCommand` is a general purpose command,  and it may be not clear that it skips plugins or themes internally.

I think it would be clearer if we changed `includePlugins` to `skipPlugins` and explicitly set it in all places which need to skip plugins. It would be also aligned with the WP CLI implementation.

Reply:
> I also thought in adding this arguments at execute-wp-cli level, but that way we would adding the arguments also when we execute wp-cli from the CLI.

Could you elaborate on this, @sejas?

Reply:
> I think it would be clearer if we changed includePlugins to skipPlugins and explicitly set it in all places which need to skip plugins. It would be also aligned with the WP CLI implementation.

Agreed! More often than not, magical code behavior produces more headaches.
I’ve inverted the conditions and added the flag where it’s needed (export, import, etc.).
https://github.com/Automattic/studio/pull/748/commits/3e99a7f68a33ddaf57cd119d037e1c19364a6146

Reply:
> > I also thought in adding this arguments at execute-wp-cli level, but that way we would adding the arguments also when we execute wp-cli from the CLI.

> Could you elaborate on this, @sejas?

@fredrikekelund , my comment wants to highlight that `server.executeWpCliCommand` is the right place to add this parameter. Another option would be adding the parameter down to the `[execute-wp-cli.ts`](https://github.com/Automattic/studio/blob/74f58c49334ca772cb6b63825dd4c3110b2cac0e/vendor/wp-now/src/execute-wp-cli.ts#L73) function, but that would create more confusion and wouldn't add any value.

Reply:
Thanks for the change @sejas , looks 100% clear now.